docker/circleci/Dockerfile

```diff
@@ -74,8 +90,16 @@ ENV CARGO_TARGET_X86_64_APPLE_DARWIN_LINKER=/usr/local/osxcross/target/bin/x86_6
 
 WORKDIR /go
 
+# Install Docker buildkit to support multi-arch image builds.
+ENV DOCKER_BUILDX_VERSION=0.5.1
+
+RUN mkdir -p /home/circleci/.docker/cli-plugins && \
+    curl -sfL -o /home/circleci/.docker/cli-plugins/docker-buildx \
+        https://github.com/docker/buildx/releases/download/v${DOCKER_BUILDX_VERSION}/buildx-v${DOCKER_BUILDX_VERSION}.linux-amd64 && \
+    chmod a+x /home/circleci/.docker/cli-plugins/docker-buildx
```

Comment:
The buildx utils used for multi-arch images aren't included in CircleCI's bundled docker client